### DISCUSSION TITLE
`warnIgnoredOnCompleteIfTargetDestroyed` outputs an error, not a warning.

### DISCUSSION THREAD

**Discussion by kyubuns** [KyryloKuzyk/PrimeTween#111] [General] [answered]
In the warnOnCompleteIgnored method of ReusableTween.cs, `Assert.LogError` is called, but isn't that a mistake for `Assert.LogWarning`?
The value of this log is important, but shouldn't it be a warning based on the variable name?

I understand that if I don't want to display a message, I can specify warnIgnoredOnCompleteIfTargetDestroyed = false.
However, it is often used in situations where OnComplete does not need to be called, and it is sometimes forgotten to be written.
It would be helpful if it could be made a Warning instead of an Error.

P.S. Whether it's an error or a warning makes a big difference to me, because it affects whether the Developer Console starts up automatically or whether it sends its own error report.

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey! The error is the designed behavior even though the setting is called "warn": PrimeTween is "warning" the user by displaying an error :)

This error message was a cornerstone design decision in PrimeTween and it goes by something like "PrimeTween should always warn users about potential errors and should never silence potential problems". 

In the case of destroying an object with OnComplete() callback on it, I wanted users to be aware of such pitfalls by displaying an error instead of a warning because users usually ignore warnings as non-critical. But in the case of ignoring OnComplete() callback, this can cause serious problems.

There is a `warnIfTargetDestroyed` parameter which you can set to `false` if you think that ignoring a particular OnComplete() is safe. More context: https://github.com/KyryloKuzyk/PrimeTween/discussions/4#discussioncomment-7496250
